src/stream/src/task/barrier_manager.rs

```diff
@@ -157,6 +157,10 @@ pub(super) enum LocalBarrierEvent {
         actor: ActorId,
         state: BackfillState,
     },
+    ReadBarrierMutation {
```

Comment:
May rename the event to `BarrierReceived`. In this future we can use it to monitor the inflight barriers in the actor.

Reply:
Actually the `ReadBarrierMutation` event is issued by actor, which is trying to read mutation from local cache. `BarrierReceived` sounds a little bit confusing here. 🤔